vstgui/lib/controls/cknob.h

```diff
@@ -33,6 +33,9 @@ class CKnobBase : public CControl, protected CMouseWheelEditingSupport
 
 	virtual CCoord getInsetValue () const { return inset; }
 	virtual void setInsetValue (CCoord val) { inset = val; }
+
+	virtual void setKnobRange (float val) { if(val > 0.f) knobRange = val; }
```

Comment:
There should be a space between the `if` and `(` ;-)

```suggestion
	virtual void setKnobRange (float val) { if (val > 0.f) knobRange = val; }
```

Reply:
@raygit83 this one I meant where the code style was wrong ;-)